molecule/common/Containerfile.j2

```diff
@@ -9,4 +9,8 @@ RUN dnf upgrade -y && \
     dnf -y install sudo python3-libselinux selinux-policy {{ item.pkg_extras | default('') }} && \
     dnf clean all -y
 
+RUN curl -o /etc/yum.repos.d/delorean.repo https://trunk.rdoproject.org/centos9-wallaby/current/delorean.repo
+
+RUN dnf install -y python*tripleo-repos
```

Comment:
I'd move the `dnf clean all -y` from the first RUN block here after the installation of `python*tripleo-repos` package.

Reply:
Yes that may be for the better.

Reply:
Agree, I will merge it into one RUN block also preserve readiness. Thanks!